<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faceDemoGuiClass</class>
 <widget class="QMainWindow" name="faceDemoGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>face_demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>581</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>450</y>
       <width>292</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectBtn">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选择图片...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="matchBtn">
        <property name="font">
         <font>
          <family>Noto Sans Mono CJK SC</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>匹配</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="selectedPic">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>480</width>
       <height>320</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addBtnTest">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>142</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Mono CJK SC</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加新人脸</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="resultGroupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>551</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>匹配结果</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>400</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="firstBtn">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="secondBtn">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="thirdBtn">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fourthBtn">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fifthBtn">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>No.5</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="matchedPicShow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>231</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>91</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Noto Sans CJK HK</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>姓名</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameShow">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Noto Sans CJK HK</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>性别</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sexShow">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Noto Sans CJK HK</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>年龄</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ageShow">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Noto Sans CJK HK</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手机</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneShow">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Noto Sans CJK HK</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>邮箱</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailShow">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSearch_Face"/>
    <addaction name="actionAdd_Face"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSearch_Face">
   <property name="text">
    <string>Search Face</string>
   </property>
  </action>
  <action name="actionAdd_Face">
   <property name="text">
    <string>Add Face</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="facedemogui.qrc"/>
 </resources>
 <connections/>
</ui>
